keithley2600/keithley_driver.py

```diff
@@ -568,6 +568,13 @@ def __init__(
 
     def __repr__(self) -> str:
         return f"<{self.__class__.__name__}({self.visa_address})>"
+        
+    def __enter__(self):
+        self.connection.write("errorqueue.clear()")
```

Comment:
Is this necessary? If yes, why not a full reset?

Reply:
wouldn't a reset bring the smu to default values? I choose the least intrusive way that allowed reliable reconnects to reconfigure the device